modules/serverless/pages/features/transport-encryption/transport-encryption-setup.adoc

```diff
@@ -0,0 +1,226 @@
+= Setup transport encryption in Eventing
+:compat-mode!:
+// Metadata:
+:description: Setup {serverlessproductname} transport encryption for Eventing
+
+This page describes the transport encryption feature which allows transporting data and events over secured and encrypted HTTPS connections using TLS.
+
+[IMPORTANT]
+====
+{serverlessproductname} transport encryption for Eventing is a Developer Preview feature only.
+Developer Preview features are not supported with Red Hat production service level agreements (SLAs) and might not be functionally complete.
+Red Hat does not recommend using them in production.
+These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information about the support scope of Red Hat Developer Preview features, see https://access.redhat.com/support/offerings/devpreview/.
+====
+
+== Prerequisites
+
+* You have access to an {product-title} account with cluster administrator access.
+
+* Install the OpenShift CLI (`oc`).
+
+* Install the {serverlessoperatorname}.
+
+* Install the {certmanageroperatorname}.
+
+== Setup a `SelfSigned` `ClusterIssuer`
+
+> `ClusterIssuers`, are Kubernetes resources that represent certificate authorities (CAs) that are able to generate signed certificates by honoring certificate signing requests.
+All cert-manager certificates require a referenced issuer that is in a ready condition to attempt to honor the request. +
+Reference: https://cert-manager.io/docs/concepts/issuer/
+
+[IMPORTANT]
+====
+For the simplicity of this guide, we will use a `SelfSigned` issuer as root certificate, however, be aware of the implications and limitations as documented at https://cert-manager.io/docs/configuration/selfsigned/ of this method.
```

Comment:
Maybe also add that we do not recommend this for production use?

Reply:
what would be our recommendation? use the same PKI as the configured OpenShift PKI or something else ?

Reply:
Highly depends on what they (the customers company) already have. For a smooth rotation, for production, at least we should recommend to use the CA issuer (where they provide their own CA as a Secret).

Reply:
Right, I think I will stay vague here and just link cert-manager docs